python/pyiceberg/catalog/base.py

```diff
@@ -187,20 +187,23 @@ def list_namespaces(self) -> list[Identifier]:
 
         Returns:
             List[Identifier]: a List of namespace identifiers
+
+        Raises:
+            NoSuchNamespaceError: If a namespace with the given name does not exist
```

Comment:
This technically doesn't accept a namespace to list yet, but it should be fine. :)